src/flux/task.coffee

```diff
@@ -169,6 +169,15 @@ TaskConfig =
 
       if allowed.indexOf(@_get(taskId).type) > -1 then true else false
 
+    getRelatedSections: (taskId) ->
+      _.chain(getSteps(@_steps[taskId]))
+        .pluck('chapter_section')
+        .compact()
+        .map( (cs) -> cs.join('.') )
+        .uniq()
+        .value()
+
+
```

Comment:
is there a way to consolidate the chapter section formatting logic either here or to the mixin?